bot/slash_commands/update.rb

```diff
@@ -0,0 +1,31 @@
+SlackRubyBotServer::Events.configure do |config|
+  config.on :command, '/update' do |command|
+    channel_name = command[:channel_name]
+    channel_id = command[:channel_id]
+    command.logger.info "Updating the incident in #{channel_name}"
```

Comment:
Is this good to log if the channel name might be invalid? Or if we want to know this in all cases, should we log the channel ID and trigger ID too?

Reply:
I guess I'm questioning whether there's any value in logging this at all?

Reply:
probably not! – will update